### PR TITLE
make project update to android gradle plugin 2.2.0

### PR DESCRIPTION
**Pull request by gengjiawen**
make project update to android gradle plugin 2.2.0
remove redundant android-apt since gradle plugin 2.2.0 provide it
update android build version
update okhttp version

I build and run it, seems work fine.
Maybe project member should add travis ci to this elegant project :)

### CODE REVIEW

**Comment by ribotaccounts** [comment]
Can one of the admins verify this patch?

**Comment by gengjiawen** [comment]
@ribotaccounts Will you conside use travis ci in this project?

**Comment by ribotaccounts** [comment]
Can one of the admins verify this patch?

File: app/build.gradle [hunk 1]
```diff
@@ -103,6 +102,7 @@ dependencies {
 
     compile 'com.squareup.sqlbrite:sqlbrite:0.5.0'
     compile "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
+    compile "com.squareup.okhttp3:okhttp:3.4.1"
```
Comment (jemmaSlater) [inline app/build.gradle]: I don't think we need this import as okhttp is not used in this boilerplate project anywhere.

File: app/build.gradle [hunk 2]
```diff
@@ -115,21 +115,21 @@ dependencies {
...
     // https://github.com/google/auto/issues/268
     provided 'com.jakewharton.auto.value:auto-value-annotations:1.2-update1'
-    apt      "com.google.auto.value:auto-value:$AUTO_VALUE_VERSION"
-    apt      'com.ryanharter.auto.value:auto-value-parcel:0.2.3-rc2'
-    apt      'com.ryanharter.auto.value:auto-value-gson:0.3.2-rc1'
-    apt      'com.squareup:javapoet:1.7.0' // https://github.com/rharter/auto-value-parcel/issues/64
+    annotationProcessor      "com.google.auto.value:auto-value:$AUTO_VALUE_VERSION"
+    annotationProcessor      'com.ryanharter.auto.value:auto-value-parcel:0.2.3-rc2'
+    annotationProcessor      'com.ryanharter.auto.value:auto-value-gson:0.3.2-rc1'
+    annotationProcessor      'com.squareup:javapoet:1.7.0' // https://github.com/rharter/auto-value-parcel/issues/64
```
Comment (jemmaSlater) [inline app/build.gradle]: Just to leave it a little tidier, we should remove this extra spacing. Previously there was extra spacing after 'apt' to improve readability by keeping the imports aligned with the 'provided' import above, but now 'annotationProcessor' takes up more space than 'provided' so the extra spacing does not serve any purpose.

**Comment by jemmaSlater** [comment]
We should already have a Jenkins tied into this project for CI, but it does not appear to be working! Will look into it.

**Comment by jemmaSlater** [comment]
I have run our tests internally and it seems fine so I'm going to merge it. :)
Will look at fixing our Jenkins setup or integrating Travis CI in the future. 
Thanks! :D

**Pull request merged**